src/wallet/wallet2.h

```diff
@@ -1458,6 +1462,7 @@ namespace tools
     cryptonote::network_type m_nettype;
     uint64_t m_kdf_rounds;
     std::string seed_language; /*!< Language of the mnemonics (seed). */
+    std::time_t creation_time;
```

Comment:
This needs to get saved to disk right? If you restore a 26 seed wallet, and then reopen it, does "dated" word remain the same as it was before? It looks like it probably gets replaced by the current time.

Have a look at `wallet2.h serialize(t_archive &a, const unsigned int ver)`